src/Xamarin.Android.Tools.Bytecode/Tests/Xamarin.Android.Tools.Bytecode-Tests.csproj

```diff
@@ -56,7 +56,13 @@
     <Compile Include="ParameterFixupTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <TestJar Include="java\**\*.java" />
+    <TestJar
+        Include="java\**\*.java"
+        Exclude="java\java\util\Collection.java"
```

Comment:
The source isn't _fully_ excluded. `Collection.java` is added to the `@(TestJarNoParameters)` item group. `@(TestJar)` is built _with_ `javac -parameters`, while `@(TestJarNoParameters)` is built _without_ `javac -parameters`.

The `Collection.java` source is still built.

Additionally, there are no other tests on `Collection.class` that check the names of parameters, so keeping it around as a file built without `javac -parameters` makes sense.